build.gradle

```
@@ -125,6 +125,9 @@ allprojects {
     ]
 
     options.errorprone {
+      if ('true'.equalsIgnoreCase(System.getProperty('avt.disableErrorProne'))) {
+        enabled = false
+      }
```

Can we do this as just:
```suggestion
      enabled = !'true'.equalsIgnoreCase(System.getProperty('avt.disableErrorProne'))
```
?